sphinx/builders/latex/constants.py

```diff
@@ -58,12 +62,50 @@
 ]
 '''
 
+XELATEX_DEFAULT_FONTPKGMATH = r'''
+\usepackage{unicode-math}
+\setmathfont{XITSMath-Regular.otf}[
+    StylisticSet=1,% choice of shape for "\mathcal"
+    BoldFont=XITSMath-Bold.otf,
+    NFSSFamily=XITS,
+]
+\makeatletter
+\AtBeginDocument{%
+  % work around unicode-math problems with \mathbf{\Gamma} et al.
+  \SetMathAlphabet{\mathrm}{normal}{TU}{XITS}{m}{n}
+  \SetMathAlphabet{\mathbf}{normal}{TU}{XITS}{b}{n}
+  \SetMathAlphabet{\mathrm}{bold}{TU}{XITS}{b}{n}
+  \SetMathAlphabet{\mathbf}{bold}{TU}{XITS}{b}{n}
+  \def\Gamma{Γ}
+  \def\Delta{Δ}
+  \def\Theta{Θ}
+  \def\Lambda{Λ}
+  \def\Xi{Ξ}
+  \def\Pi{Π}
+  \def\Sigma{Σ}
+  \def\Upsilon{Υ}
+  \def\Phi{Φ}
+  \def\Psi{Ψ}
+  \def\Omega{Ω}
+  % Make subscripts a bit larger for legibility
+  \def\defaultscriptratio{.8}
+  \def\defaultscriptscriptratio{.6}
+  \DeclareMathSizes{9}{9}{7}{5}
+  \DeclareMathSizes{\@xpt}{\@xpt}{8}{6}
+  \DeclareMathSizes{\@xipt}{\@xipt}{8.76}{6.57}
+  \DeclareMathSizes{\@xiipt}{\@xiipt}{9.6}{7.2}
+  \DeclareMathSizes{\@xivpt}{\@xivpt}{11.52}{8.64}
```

Comment:
This brings legibility benefits.  When time will come at Sphinx 9 for using `stix2` in place of `amssymb` for `pdflatex` engine, these settings will be re-used probably identically and are then completely necessary to work around some obvious rendering problems with `stix2` such as stipub/stix_type1#4.